experimental/GaloisGrp/Qt.jl

```diff
@@ -283,6 +283,16 @@ function galois_group(FF::Generic.FunctionField{fmpq}; overC::Bool = false)
   #if any fails, "t" was bad and I need to find a way of restarting
 end
 
+"""
+    subfields(FF:Generic.FunctionField{fmpq})
+
+For a finite extensino of the univariate function field over the rationals, 
+find all subfields. The implemented algorithm proceeds by substituting
+the trnascendental element to an integer, then computing the subfields
```

Comment:
```suggestion
the transcendental element to an integer, then computing the subfields
```
Wait, what is replaced by which? Unclear